function-app-deploy/azure-arm-app-service-kudu.d.ts

```diff
@@ -0,0 +1,61 @@
+import webClient = require('./webClient');

```

Comment:
you can keep the utils files in a common folder instead of having it inside both actions

Reply:
yes,, planned to publish it as node module. For now, will keep it in a common folder.